### ISSUE TITLE
Manual search

### ISSUE THREAD

**Issue by altendorfme** [morpheus65535/bazarr#237]
**Describe the bug**
Problem downloading subtitles manually

Subliminal Settings:
```
Use scene name when available: true
Minimum score for episodes: 90
Minimum score for movies: 70
Use embedded subtitles: false
Download only monitored: false
Adaptive searching: false

Enabled providers: legendastv

Single language: false
Enabled languages: Brazilian Portuguese

Movies and Series
Default enabled: true
Languages: Brazilian Portuguese
```

**To Reproduce**
Go to Movies, choice movie and click on Manually search subtitles.

```
Error: 500 Internal Server Error
Sorry, the requested URL 'https://XYZ:8443/bazarr/manual_search_movie' caused an error:

Internal Server Error
Exception:
TypeError("can't pickle BytesIO objects",)
Traceback:
Traceback (most recent call last):
  File "/opt/Bazarr/bazarr/../libs/bottle.py", line 862, in _handle
    return route.call(**args)
  File "/opt/Bazarr/bazarr/../libs/bottle.py", line 1740, in wrapper
    rv = callback(*a, **ka)
  File "/opt/Bazarr/bazarr/main.py", line 119, in wrapper
    return func(*a, **ka)
  File "/opt/Bazarr/bazarr/main.py", line 1642, in manual_search_movie_json
    data = manual_search(moviePath, language, hi, providers_list, providers_auth, sceneName, 'movie')
  File "/opt/Bazarr/bazarr/get_subtitle.py", line 238, in manual_search
    subtitles_list.append(dict(score=round((compute_score(s, video, hearing_impaired=hi) / max_score * 100), 2), language=alpha2_from_alpha3(s.language.alpha3), hearing_impaired=str(s.hearing_impaired), provider=s.provider_name, subtitle=codecs.encode(pickle.dumps(s), "base64").decode(), url=s.page_link, matches=list(matched), dont_matches=list(not_matched)))
  File "/usr/lib/python2.7/copy_reg.py", line 70, in _reduce_ex
    raise TypeError, "can't pickle %s objects" % base.__name__
TypeError: can't pickle BytesIO objects
```

**Screenshots**
![capture](https://user-images.githubusercontent.com/1530032/49033067-57c8d200-f195-11e8-9654-ff02bf48b724.PNG)

**Software (please complete the following information):**
 - Bazarr: 0.6.8
 - Radarr: 0.2.0.1217
 - Sonarr: 2.0.0.5252
 - OS: Ubuntu 16.04

**Comment by morpheus65535** [comment]
Try to force an update please. This should be fixed in 0.6.8.

**Comment by altendorfme** [comment]
In dev branch worked! thanks!

![capture](https://user-images.githubusercontent.com/1530032/49039845-15f55700-f1a8-11e8-9954-68d374b37fad.PNG)
